# Grid map
std_msgs/Header header
# all locations, including free or not
sloop_ros/GridMapLoc[] locations
